Common/DTO/NewsItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DTO
+{
+    public class NewsItem
+    {
+        public string Title { get; set; }
+
+        public string Link { get; set; }
+
+        public string PubDate { get; set; }
+
+        public string Description { get; set; }
+
+        public string Category { get; set; }
+
+        public string Author { get; set; }
+    }
+}
```

Comment:
I would add attributes [JsonPropertyName(...)] here. Who can explain it well or not?

Reply:
> I would add attributes [JsonPropertyName(...)] here. Who can explain it well or not?

Isn't it Newtonsoft.Json thing?

Reply:
no, it is system.text.json.serialize

Reply:
> no, it is system.text.json.serialize

As I understood, It's a way to explicitly say how you want properties to be named in json object, and by default they should be names just as you write in c#